postcss.config.js

```diff
@@ -18,6 +18,7 @@ module.exports = {
       core: false,
       minifyFontValues: false,
       discardUnused: false,
+      zindex: false,
```

Comment:
Don't forget to set the same settings in `site/gatsby-node.js`

Reply:
Good catch! Done 👍

Even if this setup is going to be removed soon 😅